transport_plugins/bled112/iotile_transport_bled112/bled112_cmd.py

```diff
@@ -26,6 +29,8 @@ def __init__(self, stream, commands, stop_check_interval=0.01):
         self._logger = logging.getLogger(__name__)
         self._logger.addHandler(logging.NullHandler())
         self.event_handler = None
+        self._conn_map = []
+        self._auth_manager = BLED112AuthManager(0x00, 0x01)
```

Comment:
I'm not sure its safe to save this off here.  `BLED112CommandProcessor` is an multi-ble-connection system.  What happens if we are authenticating to 2 PODS at once?  

Any state that pertains to a single connection should move to the `BLED112DeviceAdapter` code where we have a dict for keeping things on a per-connection basis.  Look, for example, at how `IOTileReportParser` is created and handled, which is also a per-connection state machine.  

Similarly, I would recommend moving `_conn_map` to `BLED112DeviceAdapter` as well.  There is already a place in the code where `BLED112DeviceAdapter` sets a `uuid` for the connection that is ultimately passed through to the AuthChain.  By having `_conn_map` in that class instead, you can just pass the correct `device_id` at that stage vs passing a connection_string to `BLED112CommandProcessor` and then having it try and convert it to a device_id before using it.